mod/wet4/views/default/wet4_theme/custom_css.php

```diff
@@ -2061,4 +2061,10 @@
 
 .wb-slc {
     height: 0;
+}
+
+.cke_focus .cke_contents {
+    border-color: #269abc;
+    outline: 0;
+    box-shadow: 0 0 3px rgba(0,0,0,.2), 0 0 3px rgba(38,154,188,.7) !important;
```

Comment:
Everithing is good and work but my suggestion will be to change  the highlight of the focus to something like that: box-shadow: 0px 0px 8px 2px rgba(102,175,233,.6), 0 0 8px rgba(102,175,233,.6).  It has the same blue has other focus and is more "perceptible".

Reply:
Sounds good to me